.github/workflows/node.js-tests.yml

```diff
@@ -95,46 +100,28 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: pnpm
 
-      - name: Set Environment variables
-        run: |
-          cp sample.env .env
-          cat .env
-
-      - name: Start MongoDB
-        uses: supercharge/mongodb-github-action@b0a1493307c4e9b82ed61f3858d606c5ff190c64 # v1.10.0
-        with:
-          mongodb-version: 6.0
-          mongodb-replica-set: test-rs
-          mongodb-port: 27017
+      - name: Set freeCodeCamp Environment Variables
+        run: cp sample.env .env
 
-      - name: Install Dependencies
+      - name: Install and Build
         run: |
-          echo pnpm version $(pnpm -v)
           pnpm install
-          pnpm run create:config
-          pnpm run build:curriculum
-
-      - name: Run Tests
-        run: pnpm test
+          pnpm run build
 
-  # DONT REMOVE THIS JOB, you dont know what it does
-  build:
-    name: Build
+  test:
+    name: Test
+    needs: build
     runs-on: ubuntu-20.04
+
     strategy:
+      fail-fast: false
```

Comment:
Is this in anticipation of having multiple node-versions? It doesn't currently do anything.

Reply:
This is pre-existing and not changed in this PR.